### ISSUE TITLE
getting issue  Can't bind to 'property' since it isn't a known property of 'tagify'. version  16.1.1 

### ISSUE THREAD

**Issue by dineshghule321** [Brakebein/ngx-tagify#58]
Hi, I’m encountering an issue: “Can't bind to 'settings' since it isn't a known property of 'tagify'.” 
This error occurs with the settings, whitelist, and readOnly properties. 
I’ve followed all the steps correctly, but the problem persists.

my app is as below versions 
![image](https://github.com/user-attachments/assets/546fb234-684f-4dab-b8a8-ee5430a8d455)

**Comment by Brakebein** [comment]
Can you share some relevant code, please?

**Comment by Brakebein** [comment]
Since you are using Angular v14, you might want to try [v14 of this library](https://www.npmjs.com/package/ngx-tagify/v/14.0.0). Because v16.1.1 is compiled against Angular v16. Apart from this, I don't see any mistakes at first sight.

**Comment by Brakebein** [comment]
Ah, the tilde import isn't supported anymore since Angular v13, but it was only fixed in this library within the last releases. Can't you just upgrade to Angular v16. Otherwise, I would need to apply this fix to this old version as well.

**Comment by Brakebein** [comment]
I just released [14.0.1](https://www.npmjs.com/package/ngx-tagify/v/14.0.1), which should resolve the issue.

If not already done, you also need to add the following to your `angular.json`:
```json
"stylePreprocessorOptions": {
  "includePaths": ["node_modules"]
},
```
